monero-outreach-docs/en/transcriptions/monero-konferenco-2019/Challenges_With_Open_Software_Revieww.md

```diff
@@ -0,0 +1,68 @@
+
+Answer: Yeah, so I actually made one, #monero-newlibs, but it's actually because of the fact that this needs to be integrated into our culture in a way. It started to occur to me that maybe it shouldn't be separate, at least not right now. There wasn't enough community awareness at the time, I think, to really make that an effective strategy. But I think that as this-- I mean, more and more people are starting to become involved in this. So I think it could actually make sense. I think that's a good idea. So that's #monero-newlibs. 
+
+There's one more back there.
+
+Question: And so I mean, it's a limited, very limited pool of people. There's a lot of contributors, but I don't think that there's very many people who have a kind of good grasp of the whole codebase, so they have a good perspective on where things where should belong, necessarily. And so, I mean, you mentioned a few people, and I was just curious if you thought--and in your response to questions, you've indicated you think it should it be a kind of group effort...and change of mindset, but I thought--I was just curious if you thought it's worth trying to convince a single person to kind of…
+
+Answer: I've already tried. I've been working on that for two years.
+
+Question: So you don't think that's the way to go, I guess?
+
+Answer: No, no. I don't think it's right either. I don't think it's right, and I don't think it's the solution either, because it's not just the case that the responsibility to factor the entire Monero codebase cannot fall on the shoulders of one individual. But it's also the case that that's not sustainable. That's not the real solution, because more people are just going to come and add code on top of their beautiful effort. So I do think it has to be like a grassroots thing. It has to be ground up. But I think, also, that the Monero codebase is actually not that crazy in terms of its functional scope and its organization. I think that if you were to, as a sort of disinterested engineer, come into wallet2 and take a look at it, you'd be able to see right off the bat that there are a whole bunch of functions that literally just can be extracted as pure functions very, very easily. And if you know a little bit about Monero, you'd be able to see, as that engineer looking at wallet2, that those are things that other people are going to want to use, so. Does that make sense? Yeah.
```

Comment:
No, no. I don't think it's right either. I don't think it's right, and I don't think it's the solution either, because it's not just the case that the responsibility to factor the entire Monero codebase cannot fall on the shoulders of one individual. But it's also the case that that's not sustainable. That's not the real solution, because more people are just going to come and add code on top of their beautiful effort. **So I do think it has to be sort of a grassroots thing.** It has to be ground up. But I think, also, that the Monero codebase is actually not that crazy in terms of its functional scope and its organization. I think that if you were to, as a sort of disinterested engineer, come into wallet2 and take a look at it, you'd be able to see right off the bat that there are a whole bunch of functions that literally just can be extracted as pure functions very, very easily. And just, if you know a little bit about Monero, you'd be able to see, as that engineer looking at wallet2, that those are things that other people **aren't** going to want to use, so. **That makes sense? Yeah.**